frontend/src/apps/device/views/find-room/index.js

```diff
@@ -56,16 +55,15 @@ const LoaderWrapper = styled.div`
 `;
 
 const AllCalendarsView = ({
-  closeAllCalendarsView,
-  calendars,
-  areAllCalendarsLoaded,
-  markUserActivity,
-  timestamp,
-  isAmPmClock,
-  fontSize
-}) => {
+                            closeAllCalendarsView,
```

Comment:
It looks like you are not using Prettier to format the code. Can you fix this PR formatting so that we can get rid of the unimportant diffs?